docs/tutorials/otio-filebundles.md

```diff
@@ -4,6 +4,18 @@
 
 This document describes OpenTimelineIO's file bundle formats, otiod and otioz.  The intent is that they make it easier to package and send or archive OpenTimelineIO data and associated media.
 
+## Source Timeline
+
+For creating otio bundles, an OTIO file is used as input, whose media references are composed only of `ExternalReference` that have a target_url field pointing at a media file with a unique basename, because file bundles have a flat namespace for media.  For example, if there are media references that point at:
+
+`/project_a/academy_leader.mov`
+
+and:
+
+`/project_b/academy_leader.mov`
+
+Because the basename of both files is `academy_leader.mov`, this will be an error.  The adapters have different policies for how to handle media references.  See below for more information.
```

Comment:
Perhaps a otioconvert could have an option to flatten, by substituting an escaped sequence for the slash?

Reply:
You mean flatten the file name in?

Reply:
yes, something like project_b___academy_leader.mov

Reply:
Yeah, I see what you're saying.  But we might also want to preserve the folder hierarchy or make some other guidelines for how files get added to the repo.  Also, this might imply that when the file is unpacked, the files would get put back somewhere new, or that the names won't be legible.  Since this adapter is speculative (at the moment), I'm inclined to keep the current (simple) restriction until we hear from use cases that call for something else.  Both this kind of a scheme and the idea of having hierarchy under the bundle wouldn't be too challenging to build.

Reply:
Works for me.